src/app/pages/CpsMap/index.jsx

```diff
@@ -1,16 +1,105 @@
 import React from 'react';
 import compose from 'ramda/src/compose';
-import CpsAssetPageMain from '../../containers/CpsAssetPageMain';
-import withContexts from '../../containers/PageHandlers/withContexts';
-import withPageWrapper from '../../containers/PageHandlers/withPageWrapper';
-import withError from '../../containers/PageHandlers/withError';
-import withLoading from '../../containers/PageHandlers/withLoading';
-import withData from '../../containers/PageHandlers/withData';
-
-const CpsMapContainer = props => {
-  return <CpsAssetPageMain {...props} />;
+import styled from 'styled-components';
+import {
+  GEL_SPACING_DBL,
+  GEL_SPACING_TRPL,
+} from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
+import { GhostGrid } from '#lib/styledGrid';
+import MetadataContainer from '#containers/Metadata';
+import LinkedData from '#containers/LinkedData';
+import headings from '#containers/Headings';
+import Timestamp from '#containers/ArticleTimestamp';
+import text from '#containers/CpsText';
+import image from '#containers/Image';
+import MediaPlayer from '#containers/CpsAssetMediaPlayer';
+import Blocks from '#containers/Blocks';
+import CpsRelatedContent from '#containers/CpsRelatedContent';
+import ATIAnalytics from '#containers/ATIAnalytics';
+import cpsAssetPagePropTypes from '../../models/propTypes/cpsAssetPage';
+import fauxHeadline from '#containers/FauxHeadline';
+import visuallyHiddenHeadline from '#containers/VisuallyHiddenHeadline';
+import { getFirstPublished } from '#containers/ArticleMain/utils';
+
+// Page Handlers
+import withContexts from '#containers/PageHandlers/withContexts';
+import withPageWrapper from '#containers/PageHandlers/withPageWrapper';
+import withError from '#containers/PageHandlers/withError';
+import withLoading from '#containers/PageHandlers/withLoading';
+import withData from '#containers/PageHandlers/withData';
+
+const CpsMapContainer = ({ pageData }) => {
+  const title = path(['promo', 'headlines', 'headline'], pageData);
+  const summary = path(['promo', 'summary'], pageData);
+  const metadata = path(['metadata'], pageData);
+  const allowDateStamp = path(['options', 'allowDateStamp'], metadata);
+  const assetUri = path(['locators', 'assetUri'], metadata);
+  const blocks = pathOr([], ['content', 'model', 'blocks'], pageData);
+  const relatedContent = pathOr(
+    [],
+    ['relatedContent', 'groups', 0, 'promos'],
+    pageData,
+  );
+  const firstPublished = getFirstPublished(pageData);
+
```

Comment:
The following code is missing: https://github.com/bbc/simorgh/blob/cf7916fc34f067dd4289eb41ca994f86138d3041/src/app/containers/CpsAssetPageMain/index.jsx#L39